# Full observation from processdata
schmalz_ecbpi/State state 	#process data in
schmalz_ecbpi/Command command	#process data out
schmalz_ecbpi/MonitorValue vacuum
schmalz_ecbpi/MonitorValue primary_voltage
schmalz_ecbpi/MonitorValue auxiliary_voltage
schmalz_ecbpi/MonitorValue temperature
uint16 evac_time_t0
uint16 evac_time_t1
uint16 leakage_rate
uint16 free_flow_vacuum
uint16 max_reached_vacuum
